Source/Core/CesiumTerrainProvider.js

```diff
@@ -200,6 +201,14 @@ define([
             }
 
             that._availableTiles = data.available;
+            for (var i = 0; i < data.minzoom; i++) {
```

Comment:
I could not find the meaning of the minzoom attribute.
I think we should remove this code and instead use empty levels in the layer.json.